rust/src/lib.rs

```diff
@@ -792,7 +792,7 @@ pub unsafe extern "C" fn zcashlc_list_transparent_receivers(
 
         // This API is documented as returning the transparent receivers for allocated
         // diversified UAs, which means change taddrs are excluded.
-        match db_data.get_transparent_receivers(account_uuid, false) {
+        match db_data.get_transparent_receivers(account_uuid, true) {
```

Comment:
If you're going to hard-code always exposing change addresses (rather than exposing the boolean as an FFI parameter), then the API documentation needs updating and every callsite on the Swift SDK side needs reviewing to ensure they all behave correctly with these new semantics.

In any case, the comment above this line (documenting why this was `false` before) needs updating.